addons/mail/static/src/components/thread_view/thread_view_tests.js

```diff
@@ -447,10 +453,12 @@ QUnit.test('new messages separator on posting message', async function (assert)
     });
     await this.createThreadViewComponent(threadViewer.threadView, { hasComposer: true });
 
-    assert.containsNone(
+    window.env = this.env;
+    window.t = threadViewer.thread;
+    assert.containsOnce(
         document.body,
         '.o_MessageList_message',
-        "should have no messages"
+        "should have the initial message"
```

Comment:
- `.o_Message` for selector
- _"should display one message in thread initially"_